monitor/internal/pod/delete_controller.go

```diff
@@ -95,38 +119,100 @@ func deleteControllerProcessItem(backgroundCtx context.Context, c client.Client,
 			// so just send a destroy event
 			if err := pc.Policy.HandlePUEvent(
 				ctx,
-				nativeID,
+				podUID,
 				common.EventDestroy,
 				policy.NewPURuntimeWithDefaults(),
 			); err != nil {
 				// we don't really care, we just warn
-				zap.L().Warn("failed to handle destroy event", zap.String("puID", nativeID), zap.String("namespacedName", req.String()), zap.Error(err))
+				zap.L().Warn("failed to handle destroy event", zap.String("puID", podUID), zap.String("namespacedName", req.String()), zap.Error(err))
```

Comment:
just writing this once here: can we prefix all `zap.L().` logs with `deleteController: ` in the `delete_controller.go` please?

some of those messages are similar to messages from the normal controller ... and it makes it easier to pinpoint